src/registry/routes/helpers/apply-default-values.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports = function(requestParameters, expectedParameters) {
+  var result = _.clone(requestParameters);
+  var optionalParametersWithDefaults = _.pick(expectedParameters, function(parameter){
+    return !parameter.mandatory && parameter.default;
```

Comment:
what when parameter.default is a boolean false? maybe a `!_.isUndefined(parameter.default)` would be better?